synapse/common.py

```diff
@@ -34,6 +34,13 @@
 import synapse.lib.msgpack as s_msgpack
 import synapse.lib.structlog as s_structlog
 
+try:
+    from yaml import CSafeLoader as Loader
+    from yaml import CSafeDumper as Dumper
+except ImportError:  # pragma: no cover
```

Comment:
Since the perf difference is significant we might want to add a log message here

Reply:
Great idea! Fixed.